src/Symfony/Component/Notifier/Exception/IncompleteDsnException.php

```diff
@@ -24,7 +24,7 @@ public function __construct(string $message, string $dsn = null, ?\Throwable $pr
     {
         $this->dsn = $dsn;
         if ($dsn) {
-            $message = sprintf('Invalid "%s" notifier DSN: ', $dsn).$message;
+            $message = sprintf('Invalid "%s" notifier DSN: %s', $dsn, $message);
```

Comment:
this was on purpose, because fabbot used to scream at `%s` that are not surrounded by double quotes.
Not sure why it doesn't scream anymore at these.
This should be double checked before merging!

Reply:
oh, I can answer myself ...:P
this is not in a `throw` expression, that's why.